src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/InvestigationSubjectTest.java

```diff
@@ -44,14 +44,14 @@ void testGetUnstructuredAddress() {
 
         investigationSubjectAddress
             .map(UnstructuredAddress::getAddressLines)
-            .contains(new String[] {"FLAT1", "12 BROWNBERRIE AVENUE", "", "LEEDS", ""});
+            .contains(new String[]{"FLAT1", "12 BROWNBERRIE AVENUE", "", "LEEDS", ""});
 
         investigationSubjectAddress
             .map(UnstructuredAddress::getFormat)
             .hasValue("US");
         investigationSubjectAddress
             .map(UnstructuredAddress::getPostCode)
-            .hasValue("LS18 5PN");
+            .hasValue(java.util.Optional.of("LS18 5PN"));
```

Comment:
Maybe a bit clearer with `.flatMap`:
```
        investigationSubjectAddress
            .flatMap(UnstructuredAddress::getPostCode)
            .contains("LS18 5PN");
```

Reply:
It definitely is! Fixed in e6ad03f86407895cb8647aff405a8ed099b0d7b8
Thanks!